acceptance/suites/pre_suite/foss/00_setup_environment.rb

```diff
@@ -1,2 +1,13 @@
-step "Initialize Test Config"
+step "Initialize Test Config" do
   PuppetServerExtensions.initialize_config options
+
+  if PuppetServerExtensions.config[:puppet_build_version] == 'LATEST'
```

Comment:
In our current [helper.rb file](https://github.com/puppetlabs/puppetserver/blob/bd05e6ef2ca0229c82526928843085bd50f0e58f/acceptance/lib/helper.rb#L28), we've been setting both the `puppet_version` and `puppet_build_version`.  Should we have something similar here for `puppet_version`?